Build failure when the `client-legacy` feature flag is enabled

Adding the following line to a new project causes a build failure:
```toml
[dependencies]
hyper-openssl = { version = "0.10.0", features = ["client-legacy"] }
```

```
> cargo check
    Checking hyper-openssl v0.10.0
error[E0432]: unresolved import `hyper_util::client::legacy::connect::HttpConnector`
  --> /home/bruce.guenter/.cargo/registry/src/index.crates.io-6f17d22bba15001f/hyper-openssl-0.10.0/src/client/legacy.rs:7:66
   |
7  | use hyper_util::client::legacy::connect::{Connected, Connection, HttpConnector};
   |                                                                  ^^^^^^^^^^^^^ no `HttpConnector` in `client::legacy::connect`
   |
note: found an item that was configured out
  --> /home/bruce.guenter/.cargo/registry/src/index.crates.io-6f17d22bba15001f/hyper-util-0.1.1/src/client/legacy/connect/mod.rs:70:22
   |
70 | pub use self::http::{HttpConnector, HttpInfo};
   |                      ^^^^^^^^^^^^^
   = note: the item is gated behind the `tokio` feature

For more information about this error, try `rustc --explain E0432`.
error: could not compile `hyper-openssl` (lib) due to previous error
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -2,7 +2,7 @@
 name = "hyper-openssl"
 version = "0.10.0"
 authors = ["Steven Fackler <sfackler@gmail.com>"]
-edition = "2018"
+edition = "2021"
 description = "Hyper TLS support via OpenSSL"
 license = "MIT/Apache-2.0"
 repository = "https://github.com/sfackler/hyper-openssl"
@@ -27,6 +27,7 @@ client-legacy = [
     "hyper-util?/client-legacy",
 ]
 
+tokio = ["hyper-util?/tokio"]
 
 [dependencies]
 http = { version = "1.0.0", optional = true }

--- a/src/client/legacy.rs
+++ b/src/client/legacy.rs
@@ -4,7 +4,9 @@ use crate::SslStream;
 use http::uri::Scheme;
 use hyper::rt::{Read, ReadBufCursor, Write};
 use hyper::Uri;
-use hyper_util::client::legacy::connect::{Connected, Connection, HttpConnector};
+#[cfg(feature = "tokio")]
+use hyper_util::client::legacy::connect::HttpConnector;
+use hyper_util::client::legacy::connect::{Connected, Connection};
 use once_cell::sync::OnceCell;
 use openssl::error::ErrorStack;
 use openssl::ex_data::Index;
@@ -115,6 +117,7 @@ pub struct HttpsConnector<T> {
     inner: Inner,
 }
 
+#[cfg(feature = "tokio")]
 impl HttpsConnector<HttpConnector> {
     /// Creates a new `HttpsConnector` using default settings.
     ///

